IPython/core/display.py

```diff
@@ -622,7 +645,7 @@ def _pngxy(data):
 def _jpegxy(data):
     """read the (width, height) from a JPEG header"""
     # adapted from http://www.64lines.com/jpeg-width-height
-    
+
```

Comment:
Ah! Thanks so much. I agree it's super annoying. I attempted to find a trick to undo it with git but couldn't find anything that worked.